### PR TITLE
Add vue-strap

### PR DESCRIPTION
**Pull request by zcyzcy88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -757,12 +757,13 @@ Tooltips / popovers
 
 ### Mobile
 
-*UI frameworks for mobile*
+*Set of components for mobile*
```
Comment (LinusBorg) [inline README.md]: Why?

File: README.md [hunk 2]
```diff
@@ -772,15 +773,11 @@ Tooltips / popovers
...
+ - [wffranco/vue-strap](https://github.com/wffranco/vue-strap) - Bootstrap 3 components built with Vue.js 2
  - [bootstrap-vue](https://bootstrap-vue.github.io) - Bootstrap 4 Components for Vue.js 2.
  - [uiv](https://github.com/wxsms/uiv) Bootstrap3 components implemented by Vue2.
 
-###### Mobile
-
-*Set of components for mobile*
-
- - [mint-ui](https://github.com/ElemeFE/mint-ui) - Mobile UI elements for Vue.js.
-
```
Comment (LinusBorg) [inline README.md]: Why did you remove this?

Comment (zcyzcy88) [inline README.md]: less redundancy, merge [`###### Mobile`](https://github.com/vuejs/awesome-vue/pull/1031/files#diff-04c6e90faac2675aa89e2176d2eec7d8L778) to [`### Mobile`](https://github.com/vuejs/awesome-vue/pull/1031/files#diff-04c6e90faac2675aa89e2176d2eec7d8R758)

()
